ci/cloudbuild/builds/cmake-install.sh

```diff
@@ -37,7 +36,7 @@ cmake "${cmake_args[@]}" \
   -DCMAKE_INSTALL_MESSAGE=NEVER \
   -DBUILD_TESTING=OFF \
   -DGOOGLE_CLOUD_CPP_ENABLE_EXAMPLES=OFF \
-  -DGOOGLE_CLOUD_CPP_ENABLE="${ENABLED_FEATURES}"
+  -DGOOGLE_CLOUD_CPP_ENABLE="__ga_libraries__,__experimental_libraries__,experimental-storage-grpc"
```

Comment:
can you add `experimental-opentelemetry` back in?

Reply:
Done